### PR TITLE
TechDebt: App Layout Fixes

### PR DESCRIPTION
**Pull request by jeznorth**
## Links to Jira Tickets

n/a

## Description of Changes

CSS Updates
Fix page layouts
Tweak some app language/error messages.

### CODE REVIEW

File: app/src/contexts/observationsContext.tsx [hunk 1]
```diff
@@ -69,9 +73,13 @@ export type IObservationsContext = {
    * API ref used to interface with an MUI DataGrid representing the observation records
    */
   _muiDataGridApiRef: React.MutableRefObject<GridApiCommunity>;
-
+  /**
+   * TODO
```
Comment (al-rosenthal) [inline app/src/contexts/observationsContext.tsx]: what are we TODO-ing here?

File: app/src/contexts/observationsContext.tsx [hunk 2]
```diff
@@ -197,5 +229,7 @@ export const ObservationsContextProvider = (props: PropsWithChildren<Record<neve
     setInitialRows
   };
 
+  console.log({ observationsContext });
```
Comment (al-rosenthal) [inline app/src/contexts/observationsContext.tsx]: nit

File: app/src/features/projects/ProjectsRouter.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { SurveyContextProvider } from 'contexts/surveyContext';
 import ProjectPage from 'features/projects/view/ProjectPage';
 import CreateSurveyPage from 'features/surveys/CreateSurveyPage';
 import SurveyRouter from 'features/surveys/SurveyRouter';
-import ProjectsLayout from 'layouts/ProjectsLayout';
+// import StandardLayout from 'layouts/StandardLayout';
```
Comment (al-rosenthal) [inline app/src/features/projects/ProjectsRouter.tsx]: nit

**Review by al-rosenthal — COMMENTED** [review]
changes look good, there are just a couple of small things to clean up

File: app/src/features/projects/ProjectsRouter.tsx (hunk 3 above)
Comment (NickPhura) [reply app/src/features/projects/ProjectsRouter.tsx]: Removed

File: app/src/contexts/observationsContext.tsx (hunk 2 above)
Comment (NickPhura) [reply app/src/contexts/observationsContext.tsx]: Removed

File: app/src/contexts/observationsContext.tsx (hunk 1 above)
Comment (NickPhura) [reply app/src/contexts/observationsContext.tsx]: I assume the jsdoc. Added some.

**Review by NickPhura — APPROVED** [review]

**Review by KjartanE — APPROVED** [review]
🐈

**Pull request merged**